healvis/simulator.py

```diff
@@ -148,13 +265,14 @@ def complete_uvdata(uv_obj, run_check=True):
     antnames = uv_obj.antenna_names     # (Nants_telescope,)
     bl_array = uv_obj.baseline_array    # (Nbls,)
     time_array = uv_obj.time_array      # (Ntimes,)
+    dt = np.diff(np.unique(time_array))[0]
 
     uv_obj.baseline_array = np.tile(bl_array, uv_obj.Ntimes)
     uv_obj.ant_1_array, uv_obj.ant_2_array = uv_obj.baseline_to_antnums(uv_obj.baseline_array)
     uv_obj.Nbls = np.unique(uv_obj.baseline_array).size
     uv_obj.Nblts = uv_obj.Nbls * uv_obj.Ntimes
     uv_obj.time_array = np.repeat(time_array, uv_obj.Nbls)
-    uv_obj.integration_time = np.repeat(np.ones_like(time_array), uv_obj.Nbls)
+    uv_obj.integration_time = np.repeat(np.ones_like(time_array), uv_obj.Nbls) * dt * 24 * 3600.    # Seconds
```

Comment:
As I said above, I think in the context of a simulator it isn't fair to set the integration time to be simply the `dt` of the `time_array`. TBH it should probably be zero or `NaN` but I think that would cause us more headache as downstream modules might break. I think setting it to one is a fair default for the time being..

Reply:
That's a very good point. In pyuvsim, we've been using integration_time as a proxy for timestep/time_cadence, but it's really the wrong term for it.

I'm going to set it to NaN, though. Setting it to 1 works, but if anything downstream uses the integration time it will introduce a silent error.

Reply:
@nkern  It seems that using NaNs causes some deep issues with the partial_freq methods (since nan != nan by python logic). We're going to discuss these parameter definitions at the datasim telecon tomorrow, if you'd like to join. For now we'll go with your idea of using 0's. At least that accurately describes the total time being integrated!

Reply:
Hmm, I guess I'm fine with that for now in this PR, but I do think that might cause some trouble later on, as I believe we have some analysis modules downstream that flags data with integration time = 0, but I guess we'll cross the bridge when we get there.